addons/web_editor/models/ir_ui_view.py

```diff
@@ -18,15 +18,15 @@
 class IrUiView(models.Model):
     _inherit = 'ir.ui.view'
 
-    def _render(self, values=None, engine='ir.qweb', minimal_qcontext=False):
+    def _render(self, values=None, engine='ir.qweb', minimal_qcontext=False, options=None):
         if values and values.get('editable'):
             try:
                 self.check_access_rights('write')
                 self.check_access_rule('write')
             except AccessError:
                 values['editable'] = False
 
-        return super(IrUiView, self)._render(values=values, engine=engine, minimal_qcontext=minimal_qcontext)
+        return super(IrUiView, self)._render(values=values, engine=engine, minimal_qcontext=minimal_qcontext, options=options)
```

Comment:
There is a problem in doing that in stable: if there is any third-party override intermediate this will break.